api/src/main/java/org/pmiops/workbench/institution/InstitutionServiceImpl.java

```diff
@@ -29,27 +32,40 @@
 
   private static final Logger log = Logger.getLogger(InstitutionServiceImpl.class.getName());
 
+  private final String OPERATIONAL_USER_INSTITUTION_SHORT_NAME = "AouOps";
```

Comment:
moved from slightly below here

Reply:
For later consideration: After experimenting in SQL  yesterday, I think we need an environment-agnostic way to refer to this special institution, or to find it. (Maybe a method we can call to get its ID for queries).

Reply:
I heard there's a plan for a flag somewhere, so that's fine.